Atomic/util.py

```diff
@@ -267,10 +173,25 @@ def skopeo(image):
     else:
         return json.loads(results.stdout.decode('utf-8'))
 
+
```

Comment:
pep8 conventions wants two blank lines between definitions not in a class.  still want me to remove.  this goes for several of the others you flagged.